CONTRIBUTING.md

```diff
@@ -461,7 +461,7 @@ Much of the management plane SDK code is generated from metadata specs about the
 
 ## Versioning
 
-For more information on how we version see [Versioning](doc/dev/Versioning.md)
+For more information on how we version see [Versioning](http://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md)
```

Comment:
nit:  I believe this should be `https`.

Reply:
Sure. Found the issue in java PR tool. Will fix it. Thanks!